src/get-started/npm-script.md

```diff
@@ -5,4 +5,48 @@ title: Get Started > NPM Script
 
 # NPM Script
 
-Need to write up.
+Use npm and `workbox-build` to build a precaching service worker. Simply install the module then cut and paste the code sample.
+
+1. [Install Node.js](https://nodejs.org/en/).
+1. Install the module with NPM.
+
+    ```
+    npm install --save-dev sw-build
+    ```
+
+1. Add the following to your `package.json`:
+
+    ```
+	"scripts": {
+	    "createsw": "node createsw.js"
+	},
+	```
+
+1. In the root of your project, create a file named `createsw.js`.
```

Comment:
Instead of instructing developers to create a standalone node script, we'd want them to provide a configuration file for `workbox-cli`. That config file might be generated as part of the first-run wizard, or it might be written by hand.